packages/common/src/errors/CustomError.ts

```diff
@@ -22,10 +22,3 @@ export class CustomError extends Error {
     }
   }
 }
-
-export class ErrorWithCause extends CustomError {
```

Comment:
Removing since there's no usage so far, we can add it later if necessary.

Reply:
Right, PascalCase files typically contain a single React component or a class declaration, and camelCase files typically contain a single React hook or a function.

I know this is a matter of preference, but for better consistency and easier handling of imports, I think that named exports are better overall.

Let's say we change `packages/common/src/errors/CustomError.ts`
```diff
-export class CustomError extends Error {
+export default class CustomError extends Error {
```

we must then change `packages/common/src/index.ts` which re-exports the `CustomError` class
```diff
-export * from './errors/CustomError';
+export { default as CustomError } from './errors/CustomError';
```

to allow consumers importing `CustomError` from `common` package (index) as a named import.

If we wanted to use `CustomError` in the same package, let's say in `packages/common/src/utils/objects.ts`

```diff
-import { CustomError } from '../errors/CustomError';
+import CustomError from '../errors/CustomError';
```

I think it's easier to handle all exports as named than to handle default ones as a special case. But that's just me :smiley:

Reply:
Also, if we wanted to add more exports to e.g. `packages/common/src/errors/CustomError.ts` those would have to be named exports, and the import syntax would look like
```ts
import CustomError, { foo } from '../errors/CustomError';
```

This may cause some people to put `foo` in a separate file, even if it's closely related to `CustomError`. Conventions shouldn't cause loss of flexibility if possible.

Reply:
As I mentioned last week, it's a preference that I think has merit, but I won't "force it"... so named exports is fine.